src/DETHRACE/common/mainloop.c

```diff
@@ -388,7 +388,10 @@ void UpdateFramePeriod(tU32* pCamera_period) {
             gLast_tick_count = new_tick_count;
         }
     } else {
+        // The following makes the timer go too fast when the real frame rate is high (=low frame period)
+#ifndef BRENDER_FIX_BUGS
```

Comment:
`DETHRACE_FIX_BUGS`?

Reply:
the same behavior is also applied to the camera on line 399 `*pCamera_period = 10;` - should that be removed also?

Reply:
Perhaps, I think the camera_period is only used by the action replay.
So that's something we need to remind when we start touching that.